test/e2e/scenarios/perf/get-network-performance-measures.go

```diff
@@ -0,0 +1,84 @@
+package perf
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ritwikranjan/perf-tests/network/benchmarks/netperf/lib"
+)
+
+type GetNetworkPerformanceMeasures struct {
+	ResultTag          string
+	KubeConfigFilePath string
+	JsonOutputFile     string
+}
+
+func (v *GetNetworkPerformanceMeasures) Prevalidate() error {
+	return nil
+}
+
+func (v *GetNetworkPerformanceMeasures) Run() error {
+	results, err := lib.PerformTests(lib.TestParams{
+		Iterations:    1,
+		Image:         "ghcr.io/ritwikranjan/nptest",
+		TestNamespace: "netperf",
+		TestFrom:      13,
+		TestTo:        17,
+		JsonOutput:    true,
+		Tag:           v.ResultTag,
+		KubeConfig:    v.KubeConfigFilePath,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to get network performance measures: %v", err)
+	}
+	sourceJsonOutputFile := results[0].JsonResultFile
+	err = copyFile(sourceJsonOutputFile, v.JsonOutputFile)
```

Comment:
it seems the usage of this is effectively a move instead of a copy, if this was renamed to move, it could encompass the `deleteFile`'s below